customers/tests/test_gdpr_api.py

```diff
@@ -0,0 +1,713 @@
+                {
+                    "key": "WINTERSTORAGELEASE",
+                    "children": [
+                        {"key": "ID", "value": str(winter_storage_lease.id)},
+                        {"key": "BOAT", "value": str(boat.id)},
+                        {
+                            "key": "STATUS",
+                            "value": dict(LeaseStatus.choices)[
+                                winter_storage_lease.status
+                            ],
+                        },
+                        {"key": "ORDERS", "value": []},
+                        {"key": "COMMENT", "value": winter_storage_lease.comment},
+                        {
+                            "key": "PLACE",
+                            "value": {
+                                "key": "WINTERSTORAGEPLACE",
+                                "children": [
+                                    {
+                                        "key": "NUMBER",
+                                        "value": winter_storage_lease.place.number,
+                                    },
+                                    {
+                                        "key": "WINTER_STORAGE_SECTION",
+                                        "value": {
+                                            "key": "WINTERSTORAGESECTION",
+                                            "children": [
+                                                {
+                                                    "key": "IDENTIFIER",
+                                                    "value": (
+                                                        winter_storage_lease.place.winter_storage_section.identifier
+                                                    ),
+                                                }
+                                            ],
+                                        },
+                                    },
+                                ],
+                            },
+                        },
+                        {"key": "SECTION", "value": None},
+                        {"key": "APPLICATION", "value": None},
+                        {
+                            "key": "START_DATE",
+                            "value": winter_storage_lease.start_date.strftime(
+                                "%d-%m-%Y"
+                            ),
+                        },
+                        {
+                            "key": "END_DATE",
+                            "value": winter_storage_lease.end_date.strftime("%d-%m-%Y"),
+                        },
+                        {"key": "STICKER_NUMBER", "value": None},
+                        {"key": "STICKER_POSTED", "value": None},
+                    ],
+                }
+            ],
+        },
+        winter_storage_leases_dict,
+    )
+
+
+@override_settings(GDPR_API_DELETE_SCOPE="berths.gdprdelete")
+def test_delete_profile(rest_api_client, requests_mock, settings):
```

Comment:
I would expect there probably are situations where deleting the `CustomerProfile` is not allowed? Maybe there the existing `on_delete` work perfectly, but maybe adding a test for that kind of situation might be useful.